test/operator.jl

```diff
@@ -495,6 +495,17 @@ const sub2 = JuMP.repl[:sub2]
         anys[1] = 10
         anys[2] = 20 + x
         @test dot(floats, anys) == 10 + 40 + 2x
+            
+        # https://github.com/JuliaOpt/JuMP.jl/pull/943
+        pull943 = Model()
+        @variable(pull943, x[1 : 10^6]);
+        setvalue.(x, rand(10^6))
+        setvalue.(y, 1 : 10^6)
```

Comment:
Sorry again. I copy and pasted from a version that had other var names.
Fixed it.